build.gradle

```diff
@@ -1,8 +1,19 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath 'com.google.gms:google-services:4.3.15'
```

Comment:
Is it possible to add this to the example app's `build.gradle` so we don't include google services for everyone that consumes the library? I believe that's often a thing that need's Terms of Service requirements for companies so it could be nice to isolate if possible

Related: There's already a `buildscript` block to the example app so I'm hoping it can just be dragged and dropped in there 🙏 but if not, I don't think it's a blocker!